torch/_prims_common/wrappers.py

```diff
@@ -39,7 +39,7 @@ def _maybe_convert_to_dtype(a, dtype):
         if a.dtype != dtype:
             # NOTE: this is incorrect on the CPU
             # See https://github.com/pytorch/pytorch/issues/77553
-            return prims.convert_element_type(a, dtype)
+            return a.to(dtype)
```

Comment:
This could increase tracing time, as `to` would go through a series of redispatches, cc @chillee, @voznesenskym

Reply:
What's the motivation behind this PR?

Reply:
Backends that want only `aten` ops in the graph they are getting.

Reply:
This introduces a new stop to the decomposition train. 
Before reaching prims.view_of, it will stop at aten.alias. Export path want to get off train at aten ops.

Reply:
@SherlockNoMad imo, we should just turn `prims.convert_element_type` into an ATen op. Note that `aten.to.dtype` is  *not* a composite explicit op - it decomposes into `aten._to_copy`.

Reply:
In this PR, I will directly use aten._to_copy instead of aten._to_copy. 
Agree that prims.convert_element_type should just be something like aten.cast.

Reply:
This introduces `aten` op to `_refs` decompositions that are meant to operate on `torch.*` API. Conceptually `.to` was better, but it introduced yet another trip through dispatched ops. Probably land it with `.to` and let's see if it regresses tracing time.

Reply:
OK, I reverted back to use .to()